tests/gis_tests/backends/postgis/tests.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+
+
+@unittest.skipUnless(connection.vendor == "postgresql", "PostgreSQL specific tests")
+class TestsPostGISCreateExtension(TestCase):
+    def _extension_exists(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT 1 FROM pg_extension WHERE extname = 'postgis'")
+            return cursor.fetchone()[0] is not None
+
+    def test_extension_is_created(self):
+        """Create PostGIS extension if it does not exist."""
+        # Make sure extension is not there before we test creation
+        with connection.cursor() as cursor:
+            cursor.execute("DROP EXTENSION IF EXISTS postgis CASCADE")
+        assert not self._extension_exists()
```

Comment:
We don't use `assert ` in tests. Please use `unittest` assertions:
```suggestion
        self.assertIs(self._extension_exists(), False)
```